docs/background/index.rst

```diff
@@ -27,11 +27,53 @@ TODO
 
 Hello World.
 
+.. _bg_models:
+
+Background Models
```

Comment:
PS: the docs you put online on Google drive don't work (I get 404 page not found errors when I click on gammapy.background).
Contrary to what I said before: please don't put docs online any more! I'll simply check out your branch, built the docs locally and make inline comments on what I think should be fixed, using screenshots where needed. This works nicely.